src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/SdkContentTests/MsftToSbSdkFiles.diff

```diff
@@ -44,6 +46,14 @@ index ------------
  ./packs/Microsoft.NETCore.App.Ref/
  ./packs/Microsoft.NETCore.App.Ref/x.y.z/
  ./packs/Microsoft.NETCore.App.Ref/x.y.z/analyzers/
+@@ ------------ @@
+ ./sdk-manifests/
+ ./sdk-manifests/x.y.z/
+ ./sdk-manifests/x.y.z/
+-./sdk-manifests/x.y.z/
```

Comment:
The microsoft sdk has the following content in `sdk-manifests/`:

- 8.0.100/
- 9.0.100-preview.1/
- 9.0.100-preview7/

The source-build sdk has the following content in `sdk-manifests/`:

- 8.0.100/
- 9.0.100-preview.7/

@MiYanni - Do you know if this is expected? Is the microsoft sdk supposed to the preview 1 manifests?

Reply:
Sorry, but I don't actually know the composition of the SDK. @marcpopMSFT Does that look like it makes sense?

Reply:
This comes from [here](https://github.com/dotnet/sdk/blob/70479edb89ef1820051e951c4977f5fdb153f8e1/eng/Versions.props#L344C6-L344C21) and is used in [BundledManifests.targets](https://github.com/dotnet/sdk/blob/70479edb89ef1820051e951c4977f5fdb153f8e1/src/Installer/redist-installer/targets/BundledManifests.targets#L8-L13).

This should definitely get updated to the latest, i.e. https://www.nuget.org/packages/Microsoft.NET.Sdk.Maui.Manifest-9.0.100-preview.5/9.0.0-preview.5.24307.10#readme-body-tab

The last time this was done was in https://github.com/dotnet/installer/pull/18693.

@jonathanpeppers

Reply:
These always lag behind 1 release, as we want publicly listed packages. There is kind of a circular dependency here, .NET SDK -> MAUI -> .NET SDK. So, when .NET 9 GA, we'll probably have .NET 9 RC 2 manifests in here.

We could update it now, though: it's been a while.

Reply:
Why are we currently lagging 5 preview releases behind? Is there a process that makes sure those get updated? And what's the customer impact here?

Reply:
I'm not aware of any customer impact. I suppose you wouldn't get the latest workloads if you used the `--skip-manifest-update` switch.

To automate this, we might be able to use the same Maestro subscription that this repo will be using:
* https://github.com/dotnet/workload-versions

Reply:
@ellahathaway if the source-build SDK doesn't contain maui workloads, then the missing `9.0.100-preview.1/` folder is expected and should be ignored. Note as discussed above, the folder name will change whenever the workload manifests for maui are updated.

@jonathanpeppers can you please file an issue in dotnet/sdk? We should probably also get the manifest updated from P1 to P5.

Reply:
> @jonathanpeppers can you please file an issue in dotnet/sdk? We should probably also get the manifest updated from P1 to P5.

I just opened a PR with the builds that shipped yesterday:
* https://github.com/dotnet/sdk/pull/42077